harvard-leeds-beckett-university.csl

```diff
@@ -15,6 +15,65 @@
     <updated>2015-03-23T12:34:00+00:00</updated>
     <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
   </info>
+  <macro name="sort-term">
+    <choose>
+      <if type="entry-dictionary">
+        <choose>
+          <if variable="author">
+            <names variable="author">
+              <name initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-last="never" and="symbol"/>
+            </names>
+          </if>
+          <else>
+            <text macro="container-title"/>
+          </else>
+        </choose>
+      </if>
+      <else-if type="legal_case legislation" match="any">
+        <text macro="title"/>
+      </else-if>
+      <else-if type="bill">
+        <text macro="title"/>
+        <text macro="container-title" prefix=". " suffix=" Bill"/>
+      </else-if>
+      <else-if type="webpage">
+        <choose>
+          <if variable="author">
+            <names variable="author">
+              <name initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-last="never" and="symbol"/>
+            </names>
+          </if>
+          <else-if variable="container-title">
+            <text macro="container-title"/>
+          </else-if>
+          <else>
+            <text macro="title"/>
+          </else>
+        </choose>
+      </else-if>
+      <else-if type="song" variable="composer">
+        <names variable="composer">
+          <name initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-last="never" and="symbol"/>
+        </names>
+      </else-if>
+      <else-if variable="author">
+        <names variable="author">
+          <name initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-last="never" and="symbol"/>
+        </names>
+      </else-if>
+      <else-if variable="editor">
+        <names variable="editor">
+          <name initialize-with=". " name-as-sort-order="all" delimiter=", " delimiter-precedes-last="never" and="symbol" suffix=" ed."/>
+        </names>
+      </else-if>
```

Comment:
No matter. I've just referred to the docs and it says this:

> ‘ed.’ and ‘eds.’ are suitable abbreviations for ‘editor’ and ‘editors’

That voids the need for that extra `<locale/>` section too. I'll update with the changes.